docs/team/sohenze.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Soh Enze's Project Portfolio Page
+---
+
+- ### Overview
+- ### Summary of Contributions
+    - **Code contributed**: [RepoSense Link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=donljh&breakdown=true)
```

Comment:
You might want to change the RepoSense Link to your own link! Our links can be found here https://nus-cs2103-ay2223s1.github.io/website/admin/teamList.html!